node_modules/oae-tenants/lib/model.js

```diff
@@ -37,6 +38,7 @@ var Tenant = module.exports.Tenant = function(alias, displayName, host, opts) {
     that.countryCode = opts.countryCode;
     that.active = (opts.active !== false);
     that.isGlobalAdminServer = (opts.isGlobalAdminServer === true);
```

Comment:
We could, but the model object doesn't have access to the config, nor would we really want to require that the `Tenant` model be "initialized" with a copy of the config on startup.